### ISSUE TITLE
Pod name not refreshed after restarting Appsody project in kube

### ISSUE THREAD

**Issue by tetchel** [eclipse/codewind#2786]
When we added restart functionality to Appsody projects, we made sure the `containerID` was updated in the project metadata. But it looks like the `podName` is not updated if the project is a Kube project. 
This means that after a project restart, PFE is reporting the old pod name.

**Comment by tetchel** [comment]
This blocks remote debug for appsody since the pod name is wrong after restarting into debug mode

**Comment by tetchel** [comment]
Erin pointed out this means the container shell won't work after a restart either.

**Comment by makandre** [comment]
After #2789 container shell works after restarting.  But debug is failing to attach:

```
> Executing task: kubectl port-forward -n eclipse-codewind cw-rmapnode01-1a8462e0-8a4d-11ea-a583-5fd774698c-rpltx 5000:9229 <
Forwarding from 127.0.0.1:5000 -> 9229
Forwarding from [::1]:5000 -> 9229
Handling connection for 5000
E0429 16:57:33.017238   52132 portforward.go:400] an error occurred forwarding 5000 -> 9229: error forwarding port 9229 to pod d7df49cc5c65e90510f09f45dba78d0be87caf1e8c222cbe0a92f48c418a6367, uid : exit status 1: 2020/04/29 20:57:33 socat[107257] E connect(5, AF=2 127.0.0.1:9229, 16): Connection refused
```

**Comment by makandre** [comment]
Not sure if the pod need to expose the ports in a certain way. In Appsody's case it's the CLI that's bringing up the pod.  Attaching the `app-deploy.yaml` they generate.

[app-deploy.yaml.txt](https://github.com/eclipse/codewind/files/4554625/app-deploy.yaml.txt)

**Comment by tetchel** [comment]
opened a new issue for the new problem
